tests/dapr_tests.mk

```diff
@@ -258,7 +261,7 @@ create-test-namespace:
 	kubectl create namespace $(DAPR_TEST_NAMESPACE)
 
 delete-test-namespace:
-	kubectl delete namespace $(DAPR_TEST_NAMESPACE) aa
```

Comment:
Why? Deleting the namespace `aa` is intentional

Reply:
It is not a typo. It is deleting both the `DAPR_TEST_NAMESPACE` and `aa` namespaces.